Services/CommerceSettingsConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OrchardCore.Commerce.Abstractions;
+using OrchardCore.Entities;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Commerce.Services
+{
+    public class CommerceSettingsConfiguration : IConfigureOptions<CommerceSettings>
+    {
+        private readonly ISiteService _site;
+        //private readonly IDataProtectionProvider _dataProtectionProvider;
```

Comment:
Please delete, rather than comment out.

Reply:
Removed commented out code and squashed the commit into the previous commit.